pkg/target/kusttarget_configplugin.go

```diff
@@ -233,6 +235,26 @@ func (kt *KustTarget) configureBuiltinImageTagTransformer(
 	return
 }
 
+func (kt *KustTarget) configureBuiltinReplicaCountTransformer(
```

Comment:
So, you're not doing what all the "legacy" transformers do, which is get field configuration from tConfig. 

You're specifying the fields directly in the transformer, which is fine and how all new transformers should work (they should get their config data independently of other plugins).

So you can drop tConfig as an argument to this function.

Reply:
The reason I originally kept the config was to comply with the `transformerConfigurator` signature. I've made a modification, see my other comment.